fmdb.xcodeproj/project.pbxproj

```diff
@@ -620,14 +620,10 @@
 				};
 			};
 			buildConfigurationList = 1DEB927808733DD40010E9CD /* Build configuration list for PBXProject "fmdb" */;
-			compatibilityVersion = "Xcode 8.0";
-			developmentRegion = English;
+			compatibilityVersion = "Xcode 12.0";
```

Comment:
Bumped the compatibility since Apple requires all submissions to be built using Xcode 12 or newer as of April 21 (ref: https://developer.apple.com/news/?id=ib31uj1j)